ios/REANodesManager.m

```diff
@@ -143,8 +131,6 @@ - (void)stopUpdatingOnAnimationFrame
 
 - (void)onAnimationFrame:(CADisplayLink *)displayLink
 {
-  _currentAnimationTimestamp = displayLink.timestamp;
```

Comment:
with this change we no longer update `_currentAnimationTimestamp`, how would that prevent it from being set to `0`